wezterm-gui/src/customglyph.rs

```diff
@@ -4725,6 +4998,7 @@ impl GlyphCache {
         polys: &[Poly],
         buffer: &mut Image,
         aa: PolyAA,
+        blend_mode: Option<BlendMode>,
```

Comment:
```suggestion
        blend_mode: BlendMode,
```

as mentioned elsewhere, let's make this required; callers can pass `BlendMode::default()` if they don't care about the value?